Utils/azuretools-core/src/com/microsoft/azuretools/ActionConstants.java

```diff
@@ -40,122 +40,128 @@ public static ActionEntity parse(String action) {
-        String CREATE_TRIGGER = build(MODULE, "create-function-trigger");
-        String CREATE_PROJECT = build(MODULE, "create-function-project");
-        String CREATE = build(MODULE, "create-function-app");
-        String DEPLOY = build(MODULE, "deploy-function-app");
-        String DELETE = build(MODULE, "delete-function-app");
-        String START = build(MODULE, "start-function-app");
-        String STOP = build(MODULE, "stop-function-app");
-        String RESTART = build(MODULE, "restart-function-app");
-        String SHOW_PROPERTIES = build(MODULE, "showprop-function-app");
-        String START_STREAMING_LOG = build(MODULE, "start-streaming-log-function-app");
-        String STOP_STREAMING_LOG = build(MODULE, "stop-streaming-log-function-app");
-        String OPEN_IN_PORTAL = build(MODULE, "open-inbrowser-function-app");
+        String OPERATION_PREFIX = MODULE + SEPARATOR;
+
+        String TRIGGER = OPERATION_PREFIX + "trigger-function";
+        String ENABLE = OPERATION_PREFIX + "enable-function";
+        String DISABLE = OPERATION_PREFIX + "disable-function";
+        String RUN = OPERATION_PREFIX + "run-function-app";
+        String CREATE_TRIGGER = OPERATION_PREFIX + "create-function-trigger";
+        String CREATE_PROJECT = OPERATION_PREFIX + "create-function-project";
+        String CREATE = OPERATION_PREFIX + "create-function-app";
+        String DEPLOY = OPERATION_PREFIX + "deploy-function-app";
+        String DELETE = OPERATION_PREFIX + "delete-function-app";
+        String START = OPERATION_PREFIX + "start-function-app";
+        String STOP = OPERATION_PREFIX + "stop-function-app";
+        String RESTART = OPERATION_PREFIX + "restart-function-app";
+        String SHOW_PROPERTIES = OPERATION_PREFIX + "showprop-function-app";
+        String START_STREAMING_LOG = OPERATION_PREFIX + "start-streaming-log-function-app";
+        String STOP_STREAMING_LOG = OPERATION_PREFIX + "stop-streaming-log-function-app";
+        String OPEN_IN_PORTAL = OPERATION_PREFIX + "open-inbrowser-function-app";
     }
 
     public interface MySQL {
         String MODULE = "azure-mysql";
-
-        String CREATE = build(MODULE, buildOperationName(MODULE, "create"));
-        String DELETE = build(MODULE, buildOperationName(MODULE, "delete"));
-        String START = build(MODULE, buildOperationName(MODULE, "start"));
-        String STOP = build(MODULE, buildOperationName(MODULE, "stop"));
-        String RESTART = build(MODULE, buildOperationName(MODULE, "restart"));
-        String OPEN_IN_PORTAL = build(MODULE, buildOperationName(MODULE, "open-in-portal"));
-        String SHOW_PROPERTIES = build(MODULE, buildOperationName(MODULE, "show-properties"));
-        String CONNECT_TO_SERVER = build(MODULE, buildOperationName(MODULE, "connect-to-server"));
-        String BIND_INTO = build(MODULE, buildOperationName(MODULE, "bind-into"));
+        String OPERATION_PREFIX = MODULE + SEPARATOR + MODULE + OPERATION_SEPARATOR;
```

Comment:
why add SEPARATOR and OPERATION_SEPARATOR here?

Reply:
in order to replave with build () method. static method doesn't work for enum.